src/plugins/amqp10.js

```diff
@@ -132,7 +134,7 @@ function getAddress (link) {
 module.exports = [
   {
     name: 'amqp10',
-    file: 'lib/sender_link.js',
+    file: path.join('lib', 'sender_link.js'),
```

Comment:
The conversion should be done in the instrumenter to avoid having to do this in plugins. This also applies to all other plugins.